build/site/component-b/dev/index.html

```diff
@@ -13,7 +12,7 @@
 <header class="header">
   <nav class="navbar">
     <div class="navbar-brand">
-      <a class="navbar-item" href="http://54.236.229.254">My Antora Website</a>
+      <a class="navbar-item" href="../..">My Antora Website</a>
```

Comment:
would "/" work instead of "../../"?

Reply:
No, this is the HTML generated by Antora when it converts AsciiDocs to webpages.

/ would bring it to the root folder, but this brings it to the site folder, which likely contains another HTML file that this will direct the user to.

Reply:
@kelvinspencer is this resolved?

Reply:
@kgm78 try just using "/" in html thats the standard method of referring to the home page of a site without using the domain name itself

Reply:
This is auto-generated HTML by Antora, I have not written any HTML files. This means that when Antora is run to convert AsciiDocs to HTML web pages, this is what the HTML it generates looks like. 

I can modify the file to use "/" but because that is not how Antora auto-generated its HTML files, next time Antora is run it will override any changes done in the "build/" directory.

Reply:
This should have been resolved by ignoring the html folder @kgm78

Reply:
I just pushed a commit to update the gitignore file to ignore this directory from now on.